### PR TITLE
Expand Publish test

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -1,37 +1,59 @@
...
+          expect(enclosureLength).to.equal("40557");
+        }
+
+        cy.request({
+          url: enclosureUrl,
+          encoding: "binary",
+        }).then((response) => {
+          cy.writeFile("test.mp3", response.body, "binary");
+          cy.readFile("test.mp3", "base64").then((mp3) => {
+            expect(mp3.length).to.equal(54076);
```
Comment (cavis) [inline cypress/support/commands.ts:36]: Is this the length of the original file?  Or did you just figure out what the cdn-arranger results in, after it does all the strip-then-add-id3?

File: cypress/e2e/publish.cy.ts [hunk 2]
```diff
@@ -1,25 +1,93 @@
...
-    // });
+    cy.get("#title").type(canary);
+    cy.get("#shortDescription").type(canary);
+    cy.get("prx-button[green=1] button.green").click();
+
+    // Confirm series creation
+    // This button shows up once we've moved from /new to the series show page
+    cy.get("button.delete").should("exist");
+
+    cy.url().then((seriesUrl) => {
```
Comment (cavis) [inline cypress/e2e/publish.cy.ts:27]: No async/await?

**Review by cavis — APPROVED** [review]
👍 looks good to me

File: cypress/e2e/publish.cy.ts (hunk 2 above)
Comment (farski) [reply cypress/e2e/publish.cy.ts:27]: Cypress, as far as I can tell, can't use await in that scenario. But I didn't dig into it too much, I could have been looking at old docs/examples. But I saw zero examples of async tests.

File: cypress/support/commands.ts (hunk 1 above)
Comment (farski) [reply cypress/support/commands.ts:36]: I ran the test with a `1` and then changed it to the actual value it failed on, so it is based on the actual file coming out of Dovetail, but beyond that I don't know how much it's taking into account.

**Pull request merged**